src/plugin/analytics/Resources/modules/administration/dashboard/components/log.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import {PropTypes as T} from 'prop-types'
+
+import {ToolPage} from '#/main/core/tool/containers/page'
+import {ListData} from '#/main/app/content/list/containers/data'
+import {trans} from '#/main/app/intl/translation'
+
+import {selectors} from '#/plugin/analytics/administration/dashboard/store/selectors'
+
+const DashboardLog = (props) =>
+  <ToolPage
+    subtitle='Log'
```

Comment:
It sould use translator. We use a modified version of BazingaJsTranslation `trans(string translationKey, object placeholders = {}, string domain = 'platform')`.